### ISSUE TITLE
old LORA_RST pin definition in variants/ttgo-lora32-v21new/

### ISSUE THREAD

**Issue by VeniceInventors** [espressif/arduino-esp32#5966]
In the [pins_arduino.h](https://github.com/espressif/arduino-esp32/blob/399f4ecbb3a4cef21e2bffa37adb6190356dfb76/variants/ttgo-lora32-v21new/pins_arduino.h#L30) pin definition file, at line 30 the reset pin is set to 12 but according to [Lilygo's site](http://www.lilygo.cn/prod_view.aspx?TypeId=50060&Id=1270) and this [pinout diagram](https://forum.arduino.cc/t/identify-ttgo-lora32-v2-1-6-digital-pin-and-analog-pin/874755), the reset pin should be set to 23.

```
< #define LORA_RST    12  // GPIO14 - SX1276 RST
> #define LORA_RST    23  // GPIO23 - SX1276 RST
```
Pin 23 is also consistent with the Liligo TBeam v1.1

**Comment by atanisoft** [comment]
@tobozo can you include this update in your PR?

**Comment by tobozo** [comment]
I hope this time the CI won't need 24h to finish its job :rofl: 

![image](https://user-images.githubusercontent.com/1893754/144618321-93a92818-4ed7-4e40-aad6-51a55c908f6a.png)

**Comment by tobozo** [comment]
see PR [#5961](https://github.com/espressif/arduino-esp32/pull/5961)

**Comment by atanisoft** [comment]
@me-no-dev this one can be closed since PR #5961 has been merged.

**Comment by me-no-dev** [comment]
Thanks @atanisoft
